src/detect-pangram.js

```diff
@@ -15,7 +15,23 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const lowerPhrase = phrase.toLowerCase();
+  const alphabet = 'abcdefghijklmnopqrstuvwxyz';
+  let counter = 0;
+
+  for (let i = 0; i < alphabet.length; i++) {
+    const panagram = alphabet[i];
+
+    if (lowerPhrase.indexOf(panagram) > -1) {
+      counter++;
```

Comment:
Counter here is redundant. If one character from `alphabet` string doesn't have a match in `lowerPhrase` string the function should return `false`